### DISCUSSION TITLE
Is there a way to get the length of non speech activity ignored before the first word is detected?

### DISCUSSION THREAD

**Discussion by SahilAtWingman** [openai/whisper#492] [Q&A] [answered]
I have noticed that the timestamp of first word always starts from 0 even if there is silence or noise in the beginning of the audio file. Is there a way to fix this or a way to get the length of audio skipped in the beginning?

**Comment by eliemaalouly** [comment]
I made a script that feeds only non-silent segments from the audio file to the whisper model. You can try it, it might work for you: [transcription-vad](https://github.com/e-maalouly/Transcription-whisper_pyannote)

**Comment by jianfch** [comment, marked as answer]
You can try this method #435.
